ci-operator/step-registry/add-redhat-registry-auth/add-redhat-registry-auth-commands.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+set -e
+set -u
+set -o pipefail
+
+function run_command() {
+    local CMD="$1"
+    echo "Running Command: ${CMD}"
+    eval "${CMD}"
+}
+
+function add_redhat_registry_auth () {
+    # get the registry configures of the cluster
+    run_command "oc extract secret/pull-secret -n openshift-config --confirm --to /tmp"; ret=$?
+    if [[ $ret -eq 0 ]]; then 
+        redhat_auth_user=$(cat "/var/run/vault/mirror-registry/registry_redhat.json" | jq -r '.user')
+        redhat_auth_password=$(cat "/var/run/vault/mirror-registry/registry_redhat.json" | jq -r '.password')
+        registry_cred=`echo $redhat_auth_user:$redhat_auth_password | base64 -w 0`
+
```

Comment:
Maybe it's better to set registry url for a parameter  in function add_redhat_registry_auth, so next time it's  easy to add other registry.